screening/openg2p.json

```diff
@@ -0,0 +1,97 @@
+{
+  "name": "OpenG2P",
+  "ownership": {
+    "isOwnershipExplicit": "Yes",
+    "copyrightURL": "https://docs.openg2p.org/"
+  },
+  "dependencies": {
+    "mandatoryDepsCreateMoreRestrictions": "No",
+    "isSoftwarePltIndependent": "",
+    "pltIndependenceDesc": ""
+  },
+  "documentation": {
+    "isDocumentationAvailable": "Yes",
+    "documentationURL": [
+      "https://docs.openg2p.org/"
+    ]
+  },
+  "NonPII": {
+    "collectsNonPII": "Yes",
+    "nonPIIAccessMechanism": "Yes"
+  },
+  "privacy": {
+    "isPrivacyCompliant": "Yes",
+    "privacyComplianceList": [
+      "OpenG2P is a tool used by adopters, the OpenG2P foundation by itself does not store any PII"
+    ],
+    "isLocIntlAdherent": "Yes",
+    "locIntlComplianceList": [
+      "OpenG2P is a tool used by adopters, the OpenG2P foundation by itself does not store any PII"
+    ]
+  },
+  "standards": {
+    "supportStandards": "Yes",
+    "standardsList": [
+      "w3c verifiable credentials and others under explorations especially around handling of PII "
+    ],
+    "evidenceStandardSupport": [
+      "Work in progress especially around compliance to open standards around handling and security of PII"
+    ],
+    "implementBestPractices": "Yes",
+    "bestPracticesList": [
+      "Principals for Digital Development; the Responsible Digital Payments Guidelines by the Better Than Cash Alliance"
+    ]
+  },
+  "doNoHarm": {
+    "preventHarm": {
+      "stepsToPreventHarm": "Yes",
+      "additionalInfoMechanismProcessesPolicies": "",
+      "additionalMitigationSteps": [
+        "https://openg2p.org/"
+      ]
+    },
+    "privacyFreedom": {
+      "freedomOfExpressionRiskStrategy": "Unknown",
+      "freedomOfExpressionRiskStrategyDescrition": ""
+    },
+    "dataPrivacySecurity": {
+      "collectsPII": "No",
+      "typesOfDataCollected": [
+        ""
+      ],
+      "thirdPartyDataSharing": "Unknown",
+      "dataSharingCircumstances": [
+        ""
+      ],
+      "ensurePrivacySecurity": "Unknown",
+      "privacySecurityDescription": ""
+    },
+    "inappropriateIllegalContent": {
+      "collectStoreDistribute": "Yes",
```

Comment:
Need to clarify this whole section with project owner. I believe the answer here is `No` as they are confusing `data` with `content`. My understanding is that this project collects `data` but it does not collect and distribute `content`

Reply:
Having visited the [website](https://openg2p.org/) and relevant [documentation](https://docs.openg2p.org/) they shared I didn't find evidence pointing to content, therefore, I agree to the claim that this project collects `data` and does not collect and distribute `content`, so I have updated the fields accordingly in [commit](https://github.com/unicef/publicgoods-candidates/pull/112/commits/a7db56c683f65fdc3ed8db3a42cee52bd3e561a1)